README.md

```diff
@@ -4,10 +4,14 @@ See the full official documentation for the Human Security AWS Lambda@Edge Enfor
 
 ## Use case
 1. This repo allows you to generate templates for each Human Security AWS Lambda@Edge Enforcer lambda [Version 4](https://edocs.humansecurity.com/docs/installation-aws-lambda-edge) (and above).
-2. The lambdas are customized and allow you to:
+2. All lambdas are customized and allow you to:
    - Edit the enforcer configuration in a separate file.
    - Use async calls to fetch specific configuration values.
    - Add your custom logic to the enforcer request/response when the handler starts and before the handler finishes, and then send it to the next handler you define.
+3. The PXCombined_Enforcer_FirstParty lambda allows you to combine two lambdas (FirstParty and Enforcer lambda) in one handler function. 
```

Comment:
I would specify here that this should only be done on the `origin-request` event,  not on `viewer-request`!